index.html

```diff
@@ -87,7 +87,7 @@ <h3>Marzoval</h3>
                         <iframe width="172" height="95" src="https://www.youtube.com/embed/5oZ9epHAdm4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
                     </div>
```

Comment:
I think you should use semantic HTML to embed the video. <video></video> instead of <div class="video"></div>

Reply:
- [ ] Please review this [comment](https://github.com/dselasea/project-review-trials/pull/1#discussion_r1015351538) 📦, it is incomplete, and kindly, as the same in the last comment, explain why are you requesting changes 🎃.